cmake/TestImpactFramework/LYTestImpactFramework.cmake

```diff
@@ -392,6 +389,23 @@ function(ly_extract_target_dependencies INPUT_DEPENDENCY_LIST OUTPUT_DEPENDENCY_
     set(${OUTPUT_DEPENDENCY_LIST} ${dependencies} PARENT_SCOPE)
 endfunction()
 
+#! ly_get_target_type: retrieves the type of target (either production or test target).
+#
+# \arg:TARGET target to get the type for
+# \arg:TARGET_TYPE the retrieved target type
+function(ly_get_target_type TARGET TARGET_TYPE)
+    get_property(O3DE_ALL_TESTS_DE_NAMSPACED GLOBAL PROPERTY O3DE_ALL_TESTS_DE_NAMSPACED)
+    if(${target_name} IN_LIST O3DE_ALL_TESTS_DE_NAMSPACED)
+        set(target_type "test")
+    else()
+        set(target_type "production")
+    endif()
+
+    set(${TARGET_TYPE} ${target_type} PARENT_SCOPE)
+    set(${TYPE_META} ${type_meta} PARENT_SCOPE)
```

Comment:
Where does `TYPE_META` get set?

Reply:
Good catch, that's a hangover from some previous refactoring. Fixed 👍